tests/helpers/index.test.js

```diff
@@ -1,30 +1,52 @@
+import { fromJS } from 'immutable'
 import { getApiUri } from '../../src/misc/url'
-import { formatDate } from '../../src/misc/snippet'
+import { formatDate, getSnippetTitle } from '../../src/misc/snippet'
 
 describe('misc', () => {
-  it('should return properly formated date', () => {
-    const incomeDate = '2018-03-11T15:28:19'
-
-    expect(formatDate(incomeDate)).toEqual('11.03.2018')
+  describe('snippet related', () => {
+    const snippet = {
+      content: 'const memoizedSyntaxes = useMemo()',
+      created_at: '2019-09-29T19:23:31',
+      id: 364066,
+      syntax: 'javascript',
+      tags: ['trg', 'rtyger'],
+      title: 'some misterious title',
+      updated_at: '2020-01-01T23:28:19',
+    }
+
+    it('should return properly formated date', () => {
+      expect(formatDate(snippet.created_at)).toEqual('29.09.2019')
+    })
+
+    it('should return properly formatted date for Jan', () => {
+      expect(formatDate(snippet.updated_at)).toEqual('01.01.2020')
+    })
+
+    it('should return propper title', () => {
+      expect(getSnippetTitle(fromJS(snippet))).toEqual(snippet.title)
+    })
+
+    it('should return propper title', () => {
+      const untitled = { ...snippet, title: undefined }
+      const expected = `#${untitled.id}, Untitled`
+
+      expect(getSnippetTitle(fromJS(untitled))).toEqual(expected)
+    })
   })
 
-  it('should return properly formatted date for Jan', () => {
-    const incomeDate = '2018-01-01T23:28:19'
-
-    expect(formatDate(incomeDate)).toEqual('01.01.2018')
-  })
+  describe('getApiUri', () => {
+    it('should construct correct url with default api version', () => {
+      process.env.API_BASE_URI = '//api.xsnippet.org'
```

Comment:
I'm not sure how it works, but in Python word it would introduce a side effect: when test is finished, the global state of this env var is changed. We usually use some mock library to temporary set desired value.